author	Szymon Janc <szymon.janc@tieto.com>
commit	86cea3b353e4026682d77e3bf2f1818bce5650a8
	Mon, 26 Nov 2012 12:55:56 +0000 (14:55 +0200)

adaptername: Remove not needed empty remove callback

Remove callback is called only if it is not NULL so there is no need to
register empty callback function.

diff -ur bluez-4.99.fix.orig/plugins/adaptername.c bluez-4.99.fix/plugins/adaptername.c
--- bluez-4.99.fix.orig/plugins/adaptername.c	2012-10-17 15:15:25.270179000 -0400
+++ bluez-4.99.fix/plugins/adaptername.c	2012-11-29 08:58:21.072249708 -0500
@@ -260,14 +260,9 @@
 	return TRUE;
 }
 
-static void adaptername_remove(struct btd_adapter *adapter)
-{
-}
-
 static struct btd_adapter_driver adaptername_driver = {
 	.name	= "adaptername",
 	.probe	= adaptername_probe,
-	.remove	= adaptername_remove,
 };
 
 static int adaptername_init(void)
